Version bump to 4.5.9 (#1436)
rtMediaWP/rtMedia: readme.txt

== Upgrade Notice ==

= 4.5.9 =

> rtMedia 4.5.9, with enhancement of new class on activity page based on the uploaded media type and fixed some issues like Post Update button issue with BuddyPress Nouveau template and Issue when user uploads the media with link in comment on buddypress activity.

Reviewer: ```suggestion
rtMedia 4.5.9, with an addition of new classes in BuddPress Activity based on the uploaded media type, and other minor fixes around BuddPress Activity.
```
